critRouter.js

```diff
@@ -27,14 +41,58 @@ router.get('/:crit_id',(req, res) => {
 				username: '@' + results[0].username
 			},
 			crit: {
+                replyCrits: [],
 				id: results[0].id,
 				created_on: results[0].created_on,
-				likes: 0,
-				replies: 0,
+				likes: results[0].likes,
+				replies: results[0].replies,
 				message: results[0].message
 			}
 		};
-		res.render('crit', crit);
+        let replies_query=`SELECT 
```

Comment:
You probably want to tidy up the indentation of your query here so that it looks nice. Suggest moving SELECT down into it's own line and then starting indentation one tab over in line with your `let` assignment on line 52.

```suggestion
        let replies_query = `
             SELECT 
```

Reply:
Your `let` also is at the wrong indentation level from the line above it as well.